src/pods/generate/BaseComponentsJsFactory.ts

```diff
@@ -0,0 +1,40 @@
+import type { IComponentsManagerBuilderOptions, LogLevel } from 'componentsjs';
+import { ComponentsManager } from 'componentsjs';
+import { joinFilePath } from '../../util/PathUtil';
+import type { ComponentsJsFactory } from './ComponentsJsFactory';
+
+/**
+ * Can be used to instantiate objects using Components.js
+ */
+export class BaseComponentsJsFactory implements ComponentsJsFactory {
```

Comment:
I'll add some comments, but I originally didn't even have an interface for this since the class is so basic in what it does. I added the interface just to be consistent and in case we do ever want a more complex solution.